cudamapper/src/application_parameters.hpp

```diff
@@ -67,6 +67,7 @@ class ApplicationParameters
     int64_t max_cached_memory_bytes;
 
 private:
+    bool custom_filtering_parameter = false; // Enables override of the adaptive default for -F (i.e., -F can be set even on small readsets)
```

Comment:
Does this have to be a member variable? It could be declared in the constructor and passed to `create_input_parsers()`.

I actually think that changing filtering parameters should be done directly in constructor (or a dedicated function), not in `create_input_parsers()`. This is a consequence of properties of queries, it doesn't really have much to do with parser creation